### PR TITLE
Added some programs in Javascript

### LINKED ISSUE #166: Add codes in different languages
Add codes of simpler algorithms like linear search, etc. in more language.

A good start will be python3, golang, javascript, C#...

But anything that's known is wholeheartedly welcomed!

**Note: This issue is for adding codes, that are already there in the directory, in different languages. If you want to add different algorithms then raise an issue first and then make a pull request.**

### PR DESCRIPTION
**Pull request by AK-codingmaniac**
Closes #166
<Do NOT delete this template>
Description: Added some programs in Javascript

#### Checklist

- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] My code gave a clean output on debugging. (NO warnings/errors)

### CODE REVIEW

**Comment by AK-codingmaniac** [comment]
I just noticed that there was a need to add codes in Javascript Language so I tried to add some

**Comment by Sahil-Gupta582** [comment]
Please add a reference to the source of the code to give proper credit.

File: Sorting and Searching/Insertion_Sort/Insertion_Sort.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+//Credit:-GeeksForGeeks
```
Comment (EmperorYP7) [inline Sorting and Searching/Insertion_Sort/Insertion_Sort.js]: Apart from this, kindly provide the link to this page/code on GFG.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]

File: Sorting and Searching/Insertion_Sort/Insertion_Sort.js (hunk 1 above)
Comment (AK-codingmaniac) [reply Sorting and Searching/Insertion_Sort/Insertion_Sort.js]: I have made the changes now.

**Review by EmperorYP7 — APPROVED** [review]
LGTM

**Pull request merged**